.gitignore

```diff
@@ -105,7 +104,6 @@ htmlcov/
 nosetests.xml
 coverage.xml
 .snyk
-.python-version
```

Comment:
Are you sure you wanted to revert this as well?

Reply:
I'm not sure where `.python-version` comes from.